docs/orchestrating-elastic-stack-applications/elasticsearch/transport-settings.asciidoc

```diff
@@ -30,6 +30,8 @@ Check the https://kubernetes.io/docs/concepts/services-networking/service/#publi
 
 NOTE: When you change the `clusterIP` setting of the service, ECK deletes and re-creates the service, as `clusterIP` is an immutable field. This will cause a short network disruption, but in most cases it should not affect existing connections as the transport module uses long-lived TCP connections.
 
+NOTE: When modifying transport service settings such as `spec.ports[*].targetPort`, you'll also need to adjust the Elasticsearch `nodeSets` configuration `transport.port` for the network settings to be fully functional.
```

Comment:
I think we currently do not really support changing the transport service port. The operator assumes the default 9300 port in various places: 
* when creating the k8s service for the transport layer (can be overridden by the user)
* when creating remote cluster configurations (not overridable by the user)
* when creating the `uncast_hosts.txt` file (not overridable by the user) 
* when configuring the container ports for the Elasticsearch Pods (overridable by the user)